code/modules/modular_computers/computers/item/tablet.dm

```diff
@@ -114,12 +114,10 @@
 	.["light_on"] = borgo?.lamp_enabled
 	.["comp_light_color"] = borgo?.lamp_color
 
-//Overrides the ui_act to make the flashlight controls link to the borg instead
+//Overrides the ui_act to make the flashlight controls link to the borg instead.
+//No need to call parent, as this device is made inaccessable from anyone that shouldn't
+//be using it via other means (inaccessable to ghosts, shuts down while borg is offline, etc).
```

Comment:
Can I still use it if I'm playing on the borg, have the window open and ghost/aghost?

Reply:
Nope, the window goes out of range (eye turns orange) as soon as you become a ghost.

Reply:
That's nice. It's still not ideal to redefine the proc without calling the parent.
Can you tweak the check that breaks this at the parent level, making a redefined proc there? That would make this code less brittle.

Reply:
Flashlight setting on modular computers are now handled by calling the `toggle_flashlight()` and `set_flashlight_color()` procs. Borg tablets now override these and don't touch `ui_act()` at all.